cas-server-core-authentication/src/test/java/org/jasig/cas/authentication/FileTrustStoreSslSocketFactoryTests.java

```diff
@@ -18,7 +19,8 @@
 public class FileTrustStoreSslSocketFactoryTests {
 
     @Test
-     public void verifyTrustStoreLoadingSuccessfullyWithCertAvailable() throws Exception {
+    @Ignore
```

Comment:
I don't know. I haven't pushed it with it enabled to find out. Well, anyway, let me enable it, push it and then we'll see